integrations/docker/images/chip-cert-bins/Dockerfile

```diff
@@ -11,22 +11,22 @@ ARG ZAP_VERSION=v2023.03.23-nightly
 
 # Ensure TARGETPLATFORM is set
 RUN case ${TARGETPLATFORM} in \
-        "linux/amd64") \
-            echo "Building for linux/amd64" \
-        ;; \
-        "linux/arm64") \
-            echo "Building for linux/arm64" \
-        ;; \
-        *) \
-        if [ -z "$TARGETPLATFORM" ] ;\
-        then \
-            echo "TARGETPLATFORM not defined! Please run from buildkit (buildx)." \
-        && return 1 ;\
-        else \
-            echo "Unsupported platform ${TARGETPLATFORM}." \
-        && return 1 ;\
-        fi \
-        ;; \
+    "linux/amd64") \
```

Comment:
Previous indentation looked a bit more readable

Reply:
This must have been the restyle script, I didn't touch this manually.

Reply:
Reverted this.. maybe I ran the restyle on too many files by accident.